app/models/approval.rb

```diff
@@ -27,11 +27,15 @@ def notify_parent_approved
     if self.parent
       self.parent.child_approved!(self)
     else
-      self.proposal.partial_approve!
+      self.proposal.approve!
     end
   end
 
   def children_approved?
     self.child_approvals.where.not(status: "approved").empty?
   end
+
+  def preorder_list
```

Comment:
Not sure if I understand the method name...does this mean that it shows the approvals in order, whether or not that order is relevant?